includes/sanitizers/class-amp-core-theme-sanitizer.php

```diff
@@ -1344,7 +1334,7 @@ static function() {
 						padding-top: 0; /* Override responsive hack which is handled by AMP layout. */
 						overflow: visible;
 					}
-					.featured-content .post-thumbnail amp-img {
+					.featured-content .post-thumbnail .wp-post-image {
```

Comment:
This could be changed instead from `amp-img` to just `img`, but since the image will have the `wp-post-image` class we can target it that way instead.